lms/urls.py

```diff
@@ -608,6 +609,9 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    urlpatterns += static(
```

Comment:
It points a url path in django development server to a physical filesystem location through which the server reads and streams the file data in response to requests for the assets.
re: DEBUG - doing the above is bad practice outside of development or special cases because you don't want app server workers tied up reading/serving files - at minimum you'd configure the webserver such as nginx to serve those files, but for more scale you'd farm them out to a CDN.